templates/common/_base/files/sysctl-userfaultfd.yaml

```diff
@@ -0,0 +1,5 @@
+mode: 0644
+path: "/etc/sysctl.d/forward.conf"
```

Comment:
This has nothing to do with IP forwarding, which is the other sysctl file you must have copied from.  I think this may result in last-one-wins semantics...

IOW please use `enable-userfault-fd.conf` or so.

Reply:
I've changed the path, but probably forgot to push it!
Thanks for finding that out!

Fixed.